composer.json

```diff
@@ -13,13 +13,9 @@
     },
     "scripts": {
         "install-bats": "if [ ! -f bin/bats ] ; then git clone https://github.com/sstephenson/bats.git; mkdir -p $HOME/bin; bats/install.sh $HOME; fi",
-        "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
-        "bats": "TERMINUS_PLUGINS_DIR=.. PATH=$HOME/bin:$PATH bats -p -t tests",
+        "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer",
+        "bats": "PATH=$HOME/bin:$PATH bats -p -t tests",
         "cs": "phpcs --standard=PSR2 -n src",
```

Comment:
Does this even execute?  Wouldn't `vendor/bin` be needed in the `PATH`?

Reply:
composer searches for binaries there when invoked :)

Reply:
Also, I fixed this install-phpcs command to actually work as expected